layouts/partials/head.html

```diff
@@ -1,62 +1,85 @@
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
-    <meta name="google-site-verification" content="j-Opn93zR4YQcV5bbYN99BmNc48BT5mzd2VpzKaW9YY" />
+<meta charset="UTF-8">
```

Comment:
Why remove the indentation of these, as they’re children of `head`?

Reply:
I have no explanation, I'll make the change before submitting.